parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -253,17 +253,19 @@ if [[ $OS == $UBUNTU_OS_NAME ]]; then
 fi
 {{end}}
 
+VALIDATION_ERR=0
+
 {{- if IsHostedMaster }}
 RES=$(retrycmd_if_failure 20 1 3 nslookup ${API_SERVER_NAME})
 STS=$?
 if [[ $STS != 0 ]]; then
-  if [[ $RES == *"168.63.129.16"* ]]; then
-    exit {{GetCSEErrorCode "ERR_K8S_API_SERVER_AZURE_DNS_LOOKUP_FAIL"}}
-  else
-    exit {{GetCSEErrorCode "ERR_K8S_API_SERVER_CONN_FAIL"}}
-  fi
+    if [[ $RES == *"168.63.129.16"*  ]]; then
```

Comment:
don't understand this magic ip. what is it's significance? 
ANd shouldn't both of the errors below be an azure dns lookup fail? nslookup is never going to connect to the api server?

Reply:
the ip is Azure dns ip https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-name-resolution-for-vms-and-role-instances#considerations
The main point is to differentiate:
1) if it's reaching out to azure dns but failed it should be an internal server error (there's an issue with private clusters dns right now)
2) if it's custom dns server but not configured correctly, this should be a client error which customer should take actions.

Reply:
these would be great to have as comments int he error messages... I guess I should just memorize the azure dns ip :)